biom/tests/test_table.py

```diff
@@ -1493,6 +1493,27 @@ def test_to_dataframe_dense(self):
         obs = example_table.to_dataframe(dense=True)
         pdt.assert_frame_equal(obs, exp)
 
+    def test_to_anndata_dense(self):
+        exp = pd.DataFrame(np.array([[0.0, 1.0, 2.0], [3.0, 4.0, 5.0]]),
+                           index=['O1', 'O2'],
+                           columns=['S1', 'S2', 'S3'])
+        adata = example_table.to_anndata(dense=True)
+        pdt.assert_frame_equal(adata.transpose().to_df(), exp.astype(np.float32))
```

Comment:
anndata is `float32`?

Reply:
It is by default. I just added an option and will update this test.

Reply:
Okay cool :) no worries just curious